app/services/analytics_events.rb

```diff
@@ -529,6 +529,44 @@ def forget_all_browsers_visited
     track_event('Forget All Browsers Visited')
   end
 
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [String] exception
+  # The user was passed by manual fraud review
+  def fraud_review_passed(
+    success:,
+    errors:,
```

Comment:
1 more thing. The ticket says we want to also track the date that a user entered fraud review.

Reply:
Ah yes, let me stick that in there

Reply:
Done